core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -223,8 +223,12 @@ class ZkSecurityMigrator(zkClient: KafkaZkClient) extends Logging {
       setAclIndividually("/")
       for (path <- ZkData.SecureRootPaths) {
         debug("Going to set ACL for %s".format(path))
-        zkClient.makeSurePersistentPathExists(path)
-        setAclsRecursively(path)
+        if (path == ControllerZNode.path && !zkClient.pathExists(path)) {
+          debug("Ignoring to set ACL for %s, because it doesn't exist".format(path))
```

Comment:
nit: we can use string-interpolation
```suggestion
          debug(s"Ignoring to set ACL for $path, because it doesn't exist")
```

Reply:
You're right, but I just want to keep the code style.
Because other codes use the `format()` function.
Thanks.